compendium/test_bundles/TestBundleDSSLE1/resources/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+    "bundle.symbolic_name" : "TestBundleDSSLE1",
+    "scr" : {
+        "version" : 1,
+        "components" : [{
+            "enabled": true,
+            "immediate" : true,
+            "implementation-class": "sample::ServiceComponentDSSLE1",
+            "service" : {
+                "interfaces" : ["test::InterfaceSLE"]
```

Comment:
for both TestBundleDSSLE1 and TestBundleDSSLE2 you can probably use an existing service interface instead of creating a new one since the interface is not relevant to the test.

Reply:
now using Interface1